lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -572,7 +572,7 @@ def main():
         http_timeout=dict(type='int'),
         load_path=dict(type='path'),
         name=dict(type='str', required=True),
-        nocache=dict(type='str', default=False),
+        nocache=dict(type='bool', default=False),
```

Comment:
Ah, that makes sense.  Should I update the other boolean options to match, or just set the choices for this one to true/false?